Assets/MixedRealityToolkit/Extensions/BoundsExtensions.cs

```diff
@@ -354,54 +354,62 @@ public static void GetColliderBoundsPoints(GameObject target, List<Vector3> boun
             Collider[] colliders = target.GetComponentsInChildren<Collider>();
             for (int i = 0; i < colliders.Length; i++)
             {
-                if (ignoreLayers == (1 << colliders[i].gameObject.layer | ignoreLayers))
-                {
-                    continue;
-                }
+                GetColliderBoundsPoints(colliders[i], boundsPoints, ignoreLayers);
+            }
+        }
 
-                if(colliders[i] is SphereCollider)
-                {
-                    SphereCollider sc = colliders[i] as SphereCollider;
-                    Bounds sphereBounds = new Bounds(sc.center, Vector3.one * sc.radius * 2);
-                    sphereBounds.GetFacePositions(sc.transform, ref corners);
-                    boundsPoints.AddRange(corners);
-                }
-                else if(colliders[i] is BoxCollider)
-                {
-                    BoxCollider bc = colliders[i] as BoxCollider;
-                    Bounds boxBounds = new Bounds(bc.center, bc.size);
-                    boxBounds.GetCornerPositions(bc.transform, ref corners);
-                    boundsPoints.AddRange(corners);
+        /// <summary>
+        /// Method to get bounds from a single Collider
+        /// </summary>
+        /// <param name="collider">Target collider</param>
+        /// <param name="boundsPoints">array reference that gets filled with points</param>
+        /// <param name="ignoreLayers">layerMask to simplify search</param>
+        public static void GetColliderBoundsPoints(Collider collider, List<Vector3> boundsPoints, LayerMask ignoreLayers)
+        {
+            if (ignoreLayers == (1 << collider.gameObject.layer | ignoreLayers)) { return; }
 
-                }
-                else if(colliders[i] is MeshCollider)
+            if (collider is SphereCollider)
+            {
+                SphereCollider sc = collider as SphereCollider;
+                Bounds sphereBounds = new Bounds(sc.center, Vector3.one * sc.radius * 2);
+                sphereBounds.GetFacePositions(sc.transform, ref corners);
+                boundsPoints.AddRange(corners);
+            }
+            else if (collider is BoxCollider)
+            {
+                BoxCollider bc = collider as BoxCollider;
+                Bounds boxBounds = new Bounds(bc.center, bc.size);
+                boxBounds.GetCornerPositions(bc.transform, ref corners);
+                boundsPoints.AddRange(corners);
+
```

Comment:
nit: extra line

Reply:
what does nit mean?

Reply:
It just means a small comment on something that's not super important but might make things more readable.